staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/structural.go

```diff
@@ -66,6 +66,8 @@ type Extensions struct {
 	// in the validation schema. This affects fields recursively,
 	// but switches back to normal pruning behaviour if nested
 	// properties or additionalProperties are specified in the schema.
+	// False means that the pruning behaviour is inherited from the parent.
+	// False does not mean to activate pruning.
 	XPreserveUnknownFields bool
```

Comment:
am undecided whether this is a good idea. In the structural schema we never want to see the nil case. Hence, I define the false differently.